cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/Enum.java

```diff
@@ -0,0 +1,18 @@
+package com.telefonica.iot.cygnus.backends.sql;
```

Comment:
License header in all new files, pls :)

Reply:
Fixed in [8ef7b73](https://github.com/telefonicaid/fiware-cygnus/pull/2007/commits/8ef7b7341de12b4112270de5291166d7e6a49b80)